KalturaWowzaServer/src/main/java/com/kaltura/media_server/modules/LiveStreamSettingsModule.java

```diff
@@ -0,0 +1,243 @@
+			}
+
+		}
+
+		public void onFillChunkEnd(LiveStreamPacketizerCupertinoChunk chunk, long timecode) {
+			logger.info("[" + chunk.getRendition().toString() + ":" + liveStreamPacketizer.getContextStr() + "]: chunkId:" + chunk.getChunkIndexForPlaylist());
+		}
+
+		public void onFillChunkMediaPacket(LiveStreamPacketizerCupertinoChunk chunk, CupertinoPacketHolder holder, AMFPacket packet) {
+			logger.info("[" + chunk.getRendition().toString() + ":" + liveStreamPacketizer.getContextStr() + "]: chunkId:" + chunk.getChunkIndexForPlaylist());
+		}
+
+		public void onFillChunkDataPacket(LiveStreamPacketizerCupertinoChunk chunk, CupertinoPacketHolder holder, AMFPacket packet, ID3Frames id3Frames) {
+			logger.info("[" + chunk.getRendition().toString() + ":" + liveStreamPacketizer.getContextStr() + "]: chunkId:" + chunk.getChunkIndexForPlaylist());
+		}
+
+	}
+
+	class LiveStreamPacketizerListener implements ILiveStreamPacketizerActionNotify {
+
+		private IApplicationInstance appInstance = null;
+
+		public LiveStreamPacketizerListener(IApplicationInstance appInstance) {
+			logger.debug("creating new LiveStreamPacketizerListener");
+			this.appInstance = appInstance;
+		}
+
+		public void onLiveStreamPacketizerCreate(ILiveStreamPacketizer liveStreamPacketizer, String streamName) {
+			logger.debug("onLiveStreamPacketizerCreate. stream: " + streamName);
+			if (!(liveStreamPacketizer instanceof LiveStreamPacketizerCupertino))
+				return;
+
+			LiveStreamPacketizerCupertino cupertinoPacketizer = (LiveStreamPacketizerCupertino) liveStreamPacketizer;
+			IMediaStream stream = this.appInstance.getStreams().getStream(streamName);
+
+			logger.info("Create [" + streamName + "]: " + liveStreamPacketizer.getClass().getSimpleName());
+			cupertinoPacketizer.setDataHandler(new LiveStreamPacketizerDataHandler(cupertinoPacketizer, streamName));
+		}
+
+		public void onLiveStreamPacketizerDestroy(ILiveStreamPacketizer liveStreamPacketizer) {
+			String streamName = liveStreamPacketizer.getAndSetStartStream(null).getName();
+			logger.debug("(" + streamName + ") onLiveStreamPacketizerDestroy");
+		}
+
+		public void onLiveStreamPacketizerInit(ILiveStreamPacketizer liveStreamPacketizer, String streamName) {
+			logger.debug("(" + streamName + ") onLiveStreamPacketizerInit");
+		}
+	}
+
+	public void onAppStart(IApplicationInstance applicationInstance) {
+		this.liveStreamPacketizerListener = new LiveStreamPacketizerListener(applicationInstance);
+		applicationInstance.addLiveStreamPacketizerListener(this.liveStreamPacketizerListener);
+		this.maxAllowedPTSDriftMillisec = applicationInstance.getProperties().getPropertyInt(MAX_ALLOWED_PTS_DRIFT_MILLISEC, DEFAULT_MAX_ALLOWED_PTS_DRIFT_MILLISEC);
+		logger.info("Modules.LiveStreamSettingsModule application instance started");
+	}
+
+	public void onStreamCreate(IMediaStream stream) {
+		stream.addLivePacketListener(new PacketListener());
+	}
+
+	public void onStreamDestroy(IMediaStream stream) {
+	}
```

Comment:
you are calling stream.addLivePacketListener on the onStreamCreate but not removing it on onStremDestroy.